amy/consents/migrations/0006_auto_20210614_1234.py

```diff
@@ -0,0 +1,103 @@
+# Generated by Django 2.2.17 on 2021-06-14 12:34
```

Comment:
Note with this migration I'm making assumptions about what is in the database. That the consents and the options I expect to be there are there. However consents can be changed via the admin view, and this is therefore not guarenteed. I don't know what would be a path forward if that is the case.